### PR TITLE
Use domain specific type for order(#2ebg3z8)

### PR DESCRIPTION
**Pull request by rathoreprashant**
### Related Issues
Implemented: types for orders

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/types/Order.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+export interface Order {
```
Comment (adityasharma7) [inline src/types/Order.ts:1]:
```suggestion
export interface Order {
    id: String;
    name: String;
    customer: any;
    /** An array containing the items purchased in this order */
    items?: Array<OrderItem>;
    /** An array containing the groups of items purchased in this order */
    itemGroup?: Array<OrderItemGroup>;
    total?: number;
    statusId?: String;
    identifications?: Array<any>;
}
```

File: src/types/Order.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    orderId: String;
+    orderName: String;
```
Comment (ymaheshwari1) [inline src/types/Order.ts:3]:
```suggestion
    orderName: string;
```

Change `String` to `string` in all type declaration

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/types/Order.ts (hunk 1 above)
Comment (rathoreprashant) [reply src/types/Order.ts:2]: Updated sir

File: src/types/Order.ts (hunk 2 above)
Comment (rathoreprashant) [reply src/types/Order.ts]: Improved Sir

**Comment by adityasharma7** [comment]
Closing as will be done differently

**Pull request closed without merging**